src/EditorFeatures/Core/Shared/Tagging/EventSources/TaggerConstants.cs

```diff
@@ -13,27 +13,32 @@ internal static class TaggerConstants
         internal const int IdleDelay = 1500;
         internal const int NonFocusDelay = 3000;
 
-        internal static int ComputeTimeDelay(this TaggerDelay behavior, ITextBuffer textBufferOpt)
+        internal static int ComputeTimeDelayMS(this TaggerDelay behavior, ITextBuffer textBufferOpt)
```

Comment:
I'd be for just using TimeSpan, but otherwise I'd prefer if we expanded "Milliseconds" before people think MS stands for Microsoft.